src/main/java/core/org/akaza/openclinica/service/StudyParticipantServiceImpl.java

```diff
@@ -123,10 +123,10 @@
 
    
 
-    public AddParticipantResponseDTO addParticipant(AddParticipantRequestDTO addParticipantRequestDTO, UserAccountBean userAccountBean, String studyOid, String siteOid , String realm,String customerUuid, ResourceBundle textsBundle,String accessToken, String register ) {
+    public AddParticipantResponseDTO addParticipant(AddParticipantRequestDTO addParticipantRequestDTO, UserAccountBean userAccountBean, Study tenantStudy, Study tenantSite, String realm,String customerUuid, ResourceBundle textsBundle,String accessToken, String register ) {
         boolean createNewParticipant=false;
-        Study tenantStudy = studyHibDao.findByOcOID(studyOid);
-        Study tenantSite = studyHibDao.findByOcOID(siteOid);
+        String studyOid = tenantStudy.getOc_oid();
+        String siteOid = tenantSite.getOc_oid();
```

Comment:
looks like we are only using oids so we don't need to pass the whole object. Please verify

Reply:
we have sub method like isEnrollmentCapped(tenantStudy,tenantSite) , which nee study object pass in